userspace/libsinsp/parsers.cpp

```diff
@@ -442,6 +442,33 @@ void sinsp_parser::process_event(sinsp_evt *evt)
 	case PPME_SYSCALL_PRCTL_X:
 		parse_prctl_exit_event(evt);
 		break;
+	case PPME_SYSCALL_NEWFSTATAT_X:
```

Comment:
New parsers. I didn't add new methods for these small parsers.
Only parse when event are successful, and store in the event fdinfo the `dirfd` related `fdinfo` (that is already present in the thread table since it was previously added by an `open` related syscall)